### ISSUE TITLE
Pulsejet Mobile App on F-Droid

### ISSUE THREAD

**Issue by feroxib** [pulsejet/memories#852]
For wider accessibility and community outreach it's a good idea to allow downloads on multiple platforms.

Talking about a community with a lot of tinkerer's the Google Play Platform is a good start for the project but more accessibility can be created easily with direct download APK on GitHub and FDroid.
Where this would also follow the style of Nectclouds way to publish their apps.

**Comment by EVOTk** [comment]
Hello 
Should be, probably closer to the stable release


https://discord.com/channels/1100157156742926406/1100157157669879921/1159681599936413807

**Comment by feroxib** [comment]
I'm sorry.
there you go

**Comment by pulsejet** [comment]
This is surprisingly high effort. If someone has experience with this then help would be appreciated.

**Comment by feroxib** [comment]
For people interested in helping it may be beneficial to specify here what:
- high effort means
- what problems you see from your POV

**Comment by pulsejet** [comment]
The time to do all these steps https://f-droid.org/en/docs/Submitting_to_F-Droid_Quick_Start_Guide/

**Comment by p1gp1g** [comment]
You can open a PR with something like this, they will inform you for the other steps to do

```
Categories:
  - Graphics
  - Multimedia
License: Apache-2.0
AuthorName:  Varun Patil 
AuthorEmail: ??
SourceCode: https://github.com/pulsejet/memories
IssueTracker: https://github.com/pulsejet/memories/issues

AutoName: Memories

RepoType: git
Repo: https://github.com/pulsejet/memories.git

Builds:
  - versionName: '0'
    versionCode: 0

AutoUpdateMode: Version
UpdateCheckMode: Tags android/\d+\.\d+
CurrentVersion: 0
CurrentVersionCode: 0

**Comment by alomsimoy** [comment]
I've been searching for a good photos app for nextcloud forever. Just discovered Memories now. My process was looking in F-Droid for nextcloud apps. Now that I found it (by looking how to setup photoprism and finding a post of yours), I know I can use something like obtanium to get it with automatic updates, so no problem there, but If this was on F-Droid, I would've found it a lot sooner.

**Comment by fz72** [comment]
@pulsejet 
I have created a new file in f-droid repo.
https://gitlab.com/fz72/fdroiddata/-/blob/gallery.memories/metadata/gallery.memories.yml

If it`s okay, I will create a merge request.

**Comment by pulsejet** [comment]
Sure, please go ahead and link it here. I'm still very unfamiliar with the involved process here.

**Comment by fz72** [comment]
@pulsejet 
I have created a new merge request for fdroiddata: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14276

Parameter "UpdateCheckMode: Tags android/*", triggers an update in F-Droid at every new release with an android*-Tag.

For parameter updateCheckData, I have used build.gradle file, because the default for UpdateCheckMode does not work.
UpdateCheckData: android/app/build.gradle|versionCode\s(\d+)|android/app/build.gradle|versionName\s"([\d.]+)
https://f-droid.org/en/docs/Build_Metadata_Reference/#UpdateCheckMode

At the moment reproducible builds are not working.
https://f-droid.org/docs/Reproducible_Builds/

**Comment by fz72** [comment]
@pulsejet There is a question about how you build the app:

https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14276#note_1710477592

Can you please answer this question when you get back.

**Comment by mvgorcum** [comment]
In case you missed it, packaging on the fdroid repo is currently blocked because the summary in fastlane is larger than 80 characters, see https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14276#note_1734094748

**Comment by pulsejet** [comment]
Thanks to everyone's help on this, the app is now available on F-Droid
https://f-droid.org/en/packages/gallery.memories/

**Comment by llucax** [comment]
Thanks! The latest releases don't seem to be available though, only 1.6.

**Comment by pulsejet** [comment]
1.7 is now available; I think it takes a while to propagate the change
